javacutil/src/main/java/org/checkerframework/javacutil/AnnotationBuilder.java

```diff
@@ -147,6 +150,62 @@ public static AnnotationMirror fromClass(
         return res;
     }
 
+    /**
+     * Creates an {@link AnnotationMirror} given by a particular annotation class. Each element is
+     * set to its default value. Errs if any element does not have a default value.
+     *
+     * @param elements the element utilities to use
+     * @param aClass the annotation class
+     * @return an {@link AnnotationMirror} of type given type
+     */
+    public static AnnotationMirror fromClassWithDefaults(
+            Elements elements, Class<? extends Annotation> aClass) {
+        AnnotationMirror result = fromName(elements, aClass.getCanonicalName());
+        setElementsToDefaultValues(result, aClass);
+        return result;
+    }
+
+    /**
+     * Changes the annotation value for the given key of the given annotation to newValue and
+     * returns the previous value.
+     */
+    @SuppressWarnings("unchecked")
+    public static Object changeAnnotationValue(Annotation annotation, String key, Object newValue) {
+        Object handler = Proxy.getInvocationHandler(annotation);
+        Field f;
+        try {
+            f = handler.getClass().getDeclaredField("memberValues");
+        } catch (NoSuchFieldException | SecurityException e) {
+            throw new IllegalStateException(e);
+        }
+        f.setAccessible(true);
+        Map<String, Object> memberValues;
+        try {
+            memberValues = (Map<String, Object>) f.get(handler);
+        } catch (IllegalArgumentException | IllegalAccessException e) {
+            throw new IllegalStateException(e);
+        }
+        Object oldValue = memberValues.get(key);
+        if (oldValue == null || oldValue.getClass() != newValue.getClass()) {
+            throw new IllegalArgumentException();
+        }
+        memberValues.put(key, newValue);
+        return oldValue;
+    }
+
+    /**
+     * Sets the values of the elements (fields) of am to their defaults. Errs if any element does
+     * not have a default value.
+     */
+    public static void setElementsToDefaultValues(
+            AnnotationMirror am, Class<? extends Annotation> aClass) {
+        for (Method m : aClass.getDeclaredMethods()) {
+            System.out.println("  " + m);
```

Comment:
This implementation only contains debug output and doesn't implement the actual functionality.
The changes to this file seem to be work in progress and not related to this PR. Maybe they were added in error and should be in a separate PR?